### PR TITLE
[v12 Migration guide] Update component migration commands

### PR DESCRIPTION
**Pull request by sophschneider**
* Update migration guide to use safer migration commands from https://github.com/Shopify/app-bridge/pull/3183
* Fix glob paths
* Fix some copy mistakes
* Delete shadow bevel regex

co author: @chloerice commits from [here](https://github.com/Shopify/polaris/compare/update-migration-guide?expand=1) and [here](https://github.com/Shopify/polaris/pull/10911)

### CODE REVIEW

File: polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.md [hunk 1]
```diff
@@ -45,15 +45,24 @@ Not on v11 yet either? Check out our other [migration guides](https://github.com
...
 
-Follow the migration guide sections below where we have the [@shopify/polaris-migrator](/tools/polaris-migrator) commands scaffolded for you to paste into your terminal. We've defaulted the glob path in the commands to run on all `ts`/`tsx`/`scss`/`css` files, but feel free to update them to your own app's relevant path, e.g., `{app,packages}/**/*.{css,scss}`. The file extensions can be adjusted depending on what migrations you are running. For example, component migrations can be run on `*.{ts,tsx}` files while token migrations can be run on `*.{css,scss}` files.
+The [polaris-migrator](/tools/polaris-migrator) CLI commands are scaffolded for you to paste into your terminal:
+
+- Tailor the directories in the command glob paths to those relevant to your app's file structure. For example, this generic monorepo glob `**/*.{css,scss}` might need to be changed to explicitly target stylesheets in `{src}/**/*.{css,scss}` in your app.
+- Adjust the file extensions for the migrations you are running. For example, React component migrations in a TypeScript app should target `*.{,ts,tsx}` files, while token migrations should target `*.{css,scss}` files.
```
Comment (laurkim) [inline polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.md]:
```suggestion
- Adjust the file extensions for the migrations you are running. For example, React component migrations in a TypeScript app should target `*.{ts,tsx}` files, while token migrations should target `*.{css,scss}` files.
```

File: polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.md [hunk 2]
```diff
@@ -858,24 +898,33 @@ The [updated split example](/components/actions/button) can also be referenced a
 
 ### Icon
 
-**🔔 Stepped migration**: You must run the `color` -> `tone` migration before running the tone rename migrations.
+**🔔 Stepped migration**: You must run the `color` -> `tone` migration after running the tone rename migrations.
 
-#### Step 1: Replace `color` prop with `tone`
+#### Step 1: Replace `warning` tone with `caution`
```
Comment (laurkim) [inline polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.md]:
```suggestion
#### Step 1: Replace `color="warning"` with `tone="caution"`
```

File: polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.md [hunk 3]
```diff
@@ -884,35 +933,38 @@ The [updated split example](/components/actions/button) can also be referenced a
...
-+ <Icon tone="success" />
+- <Icon color="warning" />
+- <Icon tone="warning" />
++ <Icon tone="caution" />
 ```
 
 </CollapsibleDetails>
 
-#### Step 2: Replace `warning` tone with `caution`
+#### Step 2: Replace `highlight` tone with `info`
```
Comment (laurkim) [inline polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.md]:
```suggestion
#### Step 2: Replace `color="highlight"` with `tone="info"`
```

File: polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.md [hunk 4]
```diff
@@ -994,24 +1040,33 @@ Backdrop is not a pattern in the new Polaris design language. If you must use a
 
 ### Text
 
-**🔔 Stepped migration**: You must run the `color` -> `tone` migration before running the tone rename migrations.
+**🔔 Stepped migration**: You must run the `color` -> `tone` migration after running the tone rename migrations.
 
-#### Step 1: Replace `color` prop with `tone`
+#### Step 1: Replace `warning` tone with `caution`
```
Comment (laurkim) [inline polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.md]:
```suggestion
#### Step 1: Replace `color="warning"` with `tone="caution"`
```

**Review by laurkim — APPROVED** [review]
Nice 💯 

Added some suggestions but not blocking for merge.

**Review by sam-b-rose — APPROVED** [review]

File: polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.md [hunk 5]
```diff
@@ -753,10 +784,16 @@ The `Button` component has been updated to replace deprecated `connectedDisclosu
 The [updated split example](/components/actions/button) can also be referenced as an example for this manual migration.
 
 ```diff
-- <Button connectedDisclosure />
+- <Button
+-   connectedDisclosure={
+-     <Popover activator={<Button icon={ChevronDownMinor} />} />
+-   }
```
Comment (chloerice) [inline polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.md]:
```suggestion
-   connectedDisclosure={{
-     icon: ChevronDownMinor,
-   }}
```

**Pull request merged**